Cargo.toml

```
@@ -29,7 +29,8 @@ default-members = [
     "aya-log-parser",
     "aya-obj",
     "aya-tool",
-    "test/integration-test",
+    # test/integration-test is omitted; including it in this list causes `cargo test` to run its
+    # tests, and that doesn't work unless they've been build with `cargo xtask`.
```

```suggestion
    # tests, and that doesn't work outside of `cargo xtask integration-test`.
```

because it's not just the stub files that are the problem, it's permissions to make `bpf()` syscalls.

---

That's not quite true; you could use `cargo xtask build-integration-test` which will print the names of the files on stdout, and then copy them somewhere and run them with the necessary permissions.